src/main/resources/META-INF/rewrite/devcenter-starter.yml

```diff
@@ -57,3 +57,47 @@ recipeList:
   # in the Security DevCenter card.
   - io.moderne.devcenter.ReportAsSecurityIssues:
       fixRecipe: org.openrewrite.java.security.OwaspTopTen
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.moderne.devcenter.DevCenterStarterQuarkus
+displayName: DevCenter
+description: >-
+  This is a default DevCenter configuration that can be used as a starting point for your own DevCenter configuration.
+  It includes a combination of upgrades, migrations, and security fixes.
+  You can customize this configuration to suit your needs.
+  
+  For more information on how to customize your DevCenter configuration, see the [DevCenter documentation](https://docs.moderne.io/user-documentation/moderne-platform/getting-started/dev-center/).
+recipeList:
+  - io.moderne.devcenter.LibraryUpgrade:
+      cardName: Move to Quarkus 3.23.0
+      groupIdPattern: io.quarkus
+      artifactIdPattern: '*'
+      version: 3.23.0
+  - io.moderne.devcenter.JavaVersionUpgrade:
+      majorVersion: 21
+      upgradeRecipe: org.openrewrite.java.migrate.UpgradeToJava21
+  - io.moderne.devcenter.JUnitJupiterUpgrade:
+      upgradeRecipe: org.openrewrite.java.testing.junit5.JUnit4to5Migration
+  - io.moderne.devcenter.SecurityStarter
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.moderne.devcenter.DevCenterStarterParent
+displayName: DevCenter
+description: >-
+  This is a default DevCenter configuration that can be used as a starting point for your own DevCenter configuration.
+  It includes a combination of upgrades, migrations, and security fixes.
+  You can customize this configuration to suit your needs.
+  
+  For more information on how to customize your DevCenter configuration, see the [DevCenter documentation](https://docs.moderne.io/user-documentation/moderne-platform/getting-started/dev-center/).
+recipeList:
+  - io.moderne.devcenter.ParentPomUpgrade:
+      cardName: Move to spring-boot-parent 3.5.0
+      groupIdPattern: org.springframework.boot
+      artifactIdPattern: spring-boot-parent
+      version: 3.5.0
```

Comment:
Same here but for `org.openrewrite.maven.UpgradeParentVersion`

Reply:
So should we be adding a named `upgradeRecipe:` here as well then?